### ISSUE TITLE
Some queries bypass Lada Cache

### ISSUE THREAD

**Issue by spiritix** [spiritix/lada-cache#84]
After some comprehensive testing, I figured out, that since Laravel 6 there are some queries that bypass Lada Cache. This doesn't cause unexpected behavior but reduces the effect of the cache. If used with Debug Bar, the "Queries" tab must not show any queries after the second page load. All queries which are still shown bypass the cache.

**Comment by xwiz** [comment]
@spiritix How do we know it's not the debug bar that's having the error? If I remember correctly I noticed activity on redis monitor cli while the debug bar didn't seem to change behaviour, but I'll take a look at this again to verify.

**Comment by spiritix** [comment]
@xwiz First, this seems to be edge case that only occurs with very complex and rare queries. It's not like a large portion of the queries would bypass the cache. But then to your question - indeed, we'd need to verify that this is not a Debug Bar issue. Theoretically, a query that is being handled by Lada Cache is obviously not executed against the database backend but instead routed via the cache. So the Debug Bar should not be able to even access it.

**Comment by xwiz** [comment]
I agree, it may well be that it's able to capture certain queries at some other method in version 6.x which LadaCache does not override.
